CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Dot Shield
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `main`.
+2. Make your changes and commit according to [gitmoji](https://gitmoji.dev/)
+3. Create a pull request and wait for our automated systems to check everything
+
+If you want to make a change to the codebase that you are unsure about, create an issue or comment on an existing
+
+## Any contributions you make will be under the MIT Software License
+
+In short, when you submit code changes, your submissions will be understood to be under the same [Mozilla Public License](https://choosealicense.com/licenses/mpl-2.0/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/dothq-extensions/adblock/issues)
+
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/dothq-extensions/adblock/issues/new); it's that easy!
+
+## Write bug reports with detail, background, and sample code
+
+When filing a bug report please follow the template. If there is any additional information that you can provide that may be relevant (e.g. network ad blocking like pihole) we would greatly appreciate it.
+
+People _love_ thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+
+We use prettier to enforce a syntax style. We prefer:
+
+- 2 spaces for indentation rather than tabs
+- Single quotes (`'`) rather than double quotes (`"`)
+- No semicolons
+- You can try running `yarn prettier` for style unification
+
+## License
+
+By contributing, you agree that your contributions will be licensed under its MPLv2 License.
```

Comment:
Change "licensed under its MPLv2 License." to "licensed under the MPL-2.0 license."